src/js/views/createCertificate/translations/pt_br.createCertificate.i18n.json

```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "Como Gerar um CSR?",
     "csrHelpSteps": {
       "step1": "Passo 1: ",
-      "step1Text": "Execute o comando: openssl req  -newkey 'rsa:2048'  -nodes  -sha256  -keyout 'privatekey.pem'  -out 'csr.pem'  -subj '/CN=Any value'",
+      "runTheCommand": "Execute o comando",
```

Comment:
Missing `:`

The english translation has `:`
![image](https://user-images.githubusercontent.com/36794274/153885798-e19385fd-b6bb-4a04-adaf-3905ec2cb17c.png)